src/configuration/utils.ts

```diff
@@ -54,7 +54,7 @@ export interface Version {
 
 export const AVAILABLE_VERSIONS: Array<SelectableValue<Version>> = [
   {
-    label: 'OpenSearch 1.0.x',
+    label: 'OpenSearch',
```

Comment:
Changed this to just `OpenSearch` so we don't have to deal with keeping up with opensearch releases (they're on 2+ now) but i can add an Opensearch 2.0.x if we'd prefer